dashboard/test/mailers/previews/pd_teachercon1819_registration_mailer_preview.rb

```diff
@@ -7,7 +7,7 @@ def accepted
   end
 
   def withdrawn
-    Pd::Teachercon1819RegistrationMailer.teacher build_teacher_registration(:withdrawn)
+    Pd::Teachercon1819RegistrationMailer.teacher build_teacher_registration(:declined)
```

Comment:
I'd prefer this terminology be consistent. Why don't we call this email preview `declined` instead or `withdrawn`?